client/errorAccum.go

```diff
@@ -71,6 +71,11 @@ func (te *TransferErrors) AddPastError(err error, timestamp time.Time) {
 	}
 }
 
+// This resets the TransferErrors object for testing purposes
+func (te *TransferErrors) ResetErrors() {
```

Comment:
It seems this is only used by unit tests - please change to not be exported outside the module.